.github/workflows/build-and-test.yml

```diff
@@ -10,11 +10,16 @@ jobs:
   build-and-test:
     name: Build and test
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        java: [ '17', '21' ]
     steps:
       - name: Run preparatory steps
-        uses: dasch-swiss/dsp-api/.github/actions/preparation@main
+        uses: dasch-swiss/dsp-api/.github/actions/preparation@ecb4a08708255d657ed3b801c339909c4bc61a79
```

Comment:
Once the updated preparation action is available in main we can set the commit ref `ecb4a08708255d657ed3b801c339909c4bc61a79` back to the main branch.